### PR TITLE
Fixes webapp to display persons tasks on clinic pages

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/medic#5639

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Please review, @garethbowen . Thank you very much!

File: webapp/src/templates/partials/contacts_content.html [hunk 1]
```diff
@@ -105,7 +105,7 @@ <h3 translate>clinic.field.children</h3>
                 heading="child.doc.name"
                 dob="child.doc.date_of_birth"
                 primary-contact="child.isPrimaryContact"
-                task-count="child.taskCount"
+                task-count="$ctrl.selected.tasksByContact && $ctrl.selected.tasksByContact[child.doc._id]"
```
Comment (garethbowen) [inline webapp/src/templates/partials/contacts_content.html]: I don't think you need to do the undefined check - angularjs should handle this gracefully and just pass the undefined.

Comment (dianabarsan) [reply webapp/src/templates/partials/contacts_content.html]: You're right! Thanks!

**Comment by wtekeu** [comment]
Merged https://github.com/medic/medic/pull/5664 into master.

**Pull request merged**